### PR TITLE
replace mentions of 'Mender Professional' with 'hosted Mender'

### PR DESCRIPTION
**Pull request by mirzak**
As we have now introduced several plans on 'hosted.mender.io' it makes
more sence to just mention the location instead of specific plans on
that server.

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>

### CODE REVIEW

File: 07.Administration/chapter.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ taxonomy:
 # Administration
 
 Configuration and administration of Mender services.
-Use [Mender Professional](https://mender.io/products/mender-professional?target=_blank) to save time setting up and maintaining the Mender server.
+Use [hosted.mender.io](https://hosted.mender.io??target=_blank) to save time setting up and maintaining the Mender server.
```
Comment (drewmoseley) [inline 07.Administration/chapter.md:12]: Two ? characters.

File: 201.Troubleshooting/03.Mender-Client/docs.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ The message shows that the Mender client rejects the Mender server's certificate
 authority (CA).
 
 If your server is using a certificate that is signed by an official Certificate Authority, then you likely
-need to update your client's root certificate store. For example, [Mender Professional](https://mender.io/products/mender-professional?target=_blank)
+need to update your client's root certificate store. For example, [hosted.mender.io](https://hosted.mender.io??target=_blank)
```
Comment (drewmoseley) [inline 201.Troubleshooting/03.Mender-Client/docs.md:103]: Two ? characters

File: 01.Getting-started/02.On-premise-installation/docs.md [hunk 3]
```diff
@@ -10,6 +10,6 @@ For a high-level introduction to Mender and its architecture, we recommend readi
 
 This section of the documentation contains tutorials to help you set up a demo server and deploy your first update with Mender. Going from a fresh system to completing your first deployment with Mender, including server setup, should take **less than 30 minutes**.
 
-! Do not follow this getting started documentation if you are using [Mender Professional](https://mender.io/products/mender-professional?target=_blank). Instead, follow the [Quickstart guide for Raspberry Pi](../quickstart-with-raspberry-pi) or the instructions you received in the welcome email when you signed up.
+! Do not follow this getting started documentation if you are using [hosted.mender.io](https://hosted.mender.io??target=_blank). Instead, follow the [Quickstart guide for Raspberry Pi](../quickstart-with-raspberry-pi) or the instructions you received in the welcome email when you signed up.
```
Comment (drewmoseley) [inline 01.Getting-started/02.On-premise-installation/docs.md:13]: Two ? characters

File: 03.Devices/10.Update-Modules/docs.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ Refer to [further reading](#further-reading) for more details.
 
 You will need a Mender server if you are using [managed mode](../../architecture/overview#modes-of-operation).
 
-For easy setup, use [Mender Professional](https://mender.io/products/mender-professional?target=_blank) or the [on-premise demo server](../../getting-started/on-premise-installation).
+For easy setup, use [hosted.mender.io](https://hosted.mender.io??target=_blank) or the [on-premise demo server](../../getting-started/on-premise-installation).
```
Comment (drewmoseley) [inline 03.Devices/10.Update-Modules/docs.md:46]: Two ? characters

File: 04.Artifacts/10.Yocto-project/01.Building/docs.md [hunk 5]
```diff
@@ -142,7 +142,7 @@ ARTIFACTIMG_FSTYPE = "ext4"
 
 !!! The size of the disk image (`.sdimg`) should match the total size of your storage so you do not leave unused space; see [the variable MENDER_STORAGE_TOTAL_SIZE_MB](../variables#mender_storage_total_size_mb) for more information. Mender selects the file system type it builds into the disk image, which is used for initial flash provisioning, based on the `ARTIFACTIMG_FSTYPE` variable. See the [section on file system types](../../../devices/yocto-project/partition-configuration#file-system-types) for more information.
 
-!!! If you are building for **Mender Professional**, make sure to set `MENDER_SERVER_URL` and `MENDER_TENANT_TOKEN` (see the comments above).
+!!! If you are building for **[hosted.mender.io](https://hosted.mender.io??target=_blank)**, make sure to set `MENDER_SERVER_URL` and `MENDER_TENANT_TOKEN` (see the comments above).
```
Comment (drewmoseley) [inline 04.Artifacts/10.Yocto-project/01.Building/docs.md:145]: Two ? characters

File: 04.Artifacts/10.Yocto-project/01.Building/docs.md [hunk 6]
```diff
@@ -267,7 +267,7 @@ ARTIFACTIMG_FSTYPE = "ext4"
...
```
Comment (drewmoseley) [inline 04.Artifacts/10.Yocto-project/01.Building/docs.md:270]: Two ? characters

File: 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md [hunk 7]
```diff
@@ -149,7 +149,7 @@ See also the section about [state scripts](../../../artifacts/state-scripts).
 #### TenantToken
 
 A token which identifies which tenant a device belongs to. This is only relevant
-if using Mender Professional.
+if using a multi-tenant environment such as [hosted.mender.io](https://hosted.mender.io??target=_blank).
```
Comment (drewmoseley) [inline 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md:152]: Two ? characters

File: 07.Administration/02.Production-installation/docs.md [hunk 8]
```diff
@@ -11,7 +11,7 @@ taxonomy:
 <!-- AUTOMATION: execute=ORIG_DIR=$PWD; function cleanup() { set +e; cd $ORIG_DIR/mender-server/production; ./run down -v; docker volume rm mender-artifacts mender-db mender-elasticsearch-db mender-redis-db; cd $ORIG_DIR; rm -rf mender-server; } -->
 <!-- AUTOMATION: execute=trap cleanup EXIT -->
 
-!!! You can save time by using [Mender Professional](https://mender.io/products/mender-professional?target=_blank); a secure Mender server ready to use, maintained by the Mender developers.
+!!! You can save time by using [hosted.mender.io](https://hosted.mender.io??target=_blank), a secure Mender server ready to use, maintained by the Mender developers.
```
Comment (drewmoseley) [inline 07.Administration/02.Production-installation/docs.md:14]: Two ? characters

File: 07.Administration/chapter.md (hunk 1 above)
Comment (mirzak) [reply 07.Administration/chapter.md]: When you copy/paste :)

File: 01.Getting-started/01.Quickstart-with-raspberry-pi/docs.md [hunk 9]
```diff
@@ -15,10 +15,10 @@ To follow this guide, you will need the following:
 * An 8 GB or larger microSD card.
 * A Raspberry Pi [universal power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply?target=_blank) or a micro USB cable.
 * Internet connectivity for your Raspberry Pi (either Ethernet or wifi available)
-* A Mender Starter account to access the [hosted server](https://hosted.mender.io).
+* A [hosted.mender.io](https://hosted.mender.io?target=_blank) account.
 
 
-### Get a Mender Starter account
+### Get a [hosted.mender.io](https://hosted.mender.io?target=_blank) account
```
Comment (estenberg) [inline 01.Getting-started/01.Quickstart-with-raspberry-pi/docs.md:21]: I think it's a bit weird to have a link in a header? It's linked below so would remove the link here.

File: 01.Getting-started/01.Quickstart-with-raspberry-pi/docs.md [hunk 10]
```diff
@@ -68,9 +68,9 @@ pi@raspberrypi:~ $
 Keep this terminal open as we will shortly use it to install the Mender client.
 
 
-### Step 2 - Login to Mender Starter
+### Step 2 - Login to [hosted.mender.io](https://hosted.mender.io?target=_blank)
```
Comment (estenberg) [inline 01.Getting-started/01.Quickstart-with-raspberry-pi/docs.md:71]: Same here (link in header)

**Comment by mirzak** [comment]
Updated to use "hosted Mender" and remove links from headers

**Pull request merged**